applications/crossbar/src/crossbar_config.erl

```diff
@@ -25,7 +25,24 @@ autoload_modules() ->
     autoload_modules([]).
 
 autoload_modules(Default) ->
-    kapps_config:get(?CONFIG_CAT, <<"autoload_modules">>, Default).
+    Modules = kapps_config:get(?CONFIG_CAT, <<"autoload_modules">>, Default),
+    remove_versioned_modules(Modules).
+
+-spec remove_versioned_modules(binaries()) -> binaries().
+remove_versioned_modules(Modules) ->
+    lists:usort(lists:map(fun remove_module_version/1, Modules)).
+
+-spec remove_module_version(binary()) -> binary().
+remove_module_version(Module) ->
+    maybe_remove_module_version(lists:reverse(binary_to_list(Module))).
+
+-spec maybe_remove_module_version(list()) -> binary().
+maybe_remove_module_version("1v_" ++ Module) ->
+    list_to_binary(lists:reverse(Module));
+maybe_remove_module_version("2v_" ++ Module) ->
+    list_to_binary(lists:reverse(Module));
+maybe_remove_module_version(Module) ->
+    list_to_binary(lists:reverse(Module)).
```

Comment:
This way binary_to_list and list reversing stay in the same function. It makes it easier to understand.
Also it saves reading 3x `list_to_binary(lists:reverse(`